GameServ/GameServer.cpp

```diff
@@ -293,23 +294,23 @@ Game_AgeInfo age_parse(FILE* stream)
 
     Game_AgeInfo age;
     while (fgets(lnbuffer, 4096, stream)) {
-        std::vector<DS::String> line = DS::String(lnbuffer).strip('#').split('=');
+        std::vector<ST::string> line = ST::string(lnbuffer).before_first('#').trim().split('=');
```

Comment:
I'm uncertain if this is a regression or just an oversight that I'm only now noticing, but blank lines in age files trigger the `[Game] Invalid AGE line:` debug print. It's a bit confusing to see that, considering it yields a blank line.